examples/shader/shader_prepass.rs

```diff
@@ -0,0 +1,242 @@
+//! Bevy has an optional prepass that is controlled per-material. A prepass is a rendering pass that runs before the main pass.
+//! It will optionally generate various view textures. Currently it supports depth and normal textures.
+//! The textures are not generated for any material using alpha blending.
+
+use bevy::{
+    core_pipeline::prepass::{DepthPrepass, NormalPrepass},
+    pbr::{NotShadowCaster, PbrPlugin},
+    prelude::*,
+    reflect::TypeUuid,
+    render::render_resource::{AsBindGroup, ShaderRef},
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(PbrPlugin {
+            // The prepass is enabled by default on the StandardMaterial,
+            // but you can disable it if you need to.
+            //
+            // prepass_enabled: false,
+            ..default()
+        }))
+        .add_plugin(MaterialPlugin::<CustomMaterial>::default())
+        .add_plugin(MaterialPlugin::<PrepassOutputMaterial> {
+            // This material only needs to read the prepass textures
+            // but it shouldn't write to it, so we can disable it.
```

Comment:
```suggestion
            // This material only needs to read the prepass textures,
            // but the meshes using it should not contribute to the prepass render, so we can disable it.
```

Or something like this.